src/main/org/nlogo/api/VersionHistory.scala

```diff
@@ -195,6 +195,16 @@ object VersionHistory {
     remove3d(version) == "NetLogo 4.2pre3" ||
     remove3d(version) == "NetLogo 4.2pre4"
 
+  def olderThan52(version: String) =
```

Comment:
5.2 is 5.1.1, we just named it this because of the book.  Just like 5.1 was 5.0.6, named such because of the new nw extension.  5.2 is effectively 5.0.7, at least from core netlogo's point of view.